docs/blog/posts/2024/11/hpcwire_award_2024.md

```diff
@@ -0,0 +1,56 @@
+---
+authors: [boegel]
+date: 2024-11-18
+slug: hpcwire-readers-choice-awards-2024-for-eessi
+---
+
+# EESSI won an HPCWire Reader's Choice Award!
+
+We are thrilled to announce that **EESSI has won an HPCWire Reader's Choice Award**!
+
+<figure markdown="span">
+![EuroHPC User Day 2024](eessi-hpcwire-award-2024.webp){width=50%}
+</figure>
+
+<!-- more -->
+
+EESSI received the most votes from the HPC community in the *"Best HPC Programming Tool or Technology"* category,
+despite the fierce competition of others projects that got nominated in this category.
+
+This news was revealed at the [Supercomputing 2024 (SC'24) conference in Atlanta (US)](https://sc24.supercomputing.org).
+
+**Thank you very much if you voted for us!**
+
+
+## Award ceremony
+
+<figure markdown="span">
+![EuroHPC User Day 2024](eessi-hpcwire-award-2024-DoITNow-booth.webp){width=80%}
+</figure>
+
+A modest award ceremony was held at the [Do IT Now](https://www.doitnowgroup.com) booth on the SC'24 exhibit floor,
+since [HPCNow!](https://hpcnow.com) (now also known as Do IT Now Spain)
```

Comment:
```suggestion
since [HPCNow!](https://hpcnow.com) (part of the Do IT Now Group)
```
Not sure about the wording here, I looked at the [Do IT Now website](https://www.doitnowgroup.com/about/) and they say

> Do IT Systems and HPCNow! Operated independently until joining forces and created the French Joint venture Do IT Now France. Do IT Now Group is the result of the merge between Do IT Systems (Italy), HPCNow! (Spain), Do IT Now France, Do IT Now Germany and HPC Now New Zealand.

Reply:
@draentropia Can you help us cut the knot on this?

Reply:
since [HPCNow!](https://hpcnow.com) (part of the Do IT Now Group)

for now this is the good one.